prow/cmd/sinker/main.go

```diff
@@ -91,10 +102,10 @@ func main() {
 		kubeClients[alias] = kubeClient(client)
 	}
 	c := controller{
-		logger:      logrus.NewEntry(logrus.StandardLogger()),
-		kc:          kc,
-		pkcs:        kubeClients,
-		configAgent: configAgent,
+		logger:        logrus.NewEntry(logrus.StandardLogger()),
+		prowJobClient: pjclient.ProwV1().ProwJobs(configAgent.Config().ProwJobNamespace),
+		pkcs:          kubeClients,
```

Comment:
Just curious if we can also use the real client for pods instead of our kube.Client?

Reply:
It's a follow up :)

Reply:
After the other massive PR I want to make small changes and not rebase hell

Reply:
Add a TODO to remove this?

Reply:
I have the second pull already made. No need.